K2Bridge/Visitors/MetricAggregationVisitor.cs

```diff
@@ -57,5 +57,22 @@ public void Visit(SumAggregation sumAggregation)
 
             sumAggregation.KustoQL = $"['{sumAggregation.Key}']={KustoQLOperators.Sum}({sumAggregation.Field})";
         }
+
+        /// <inheritdoc/>
+        public void Visit(PercentileAggregation percentileAggregation)
+        {
+            Ensure.IsNotNull(percentileAggregation, nameof(percentileAggregation));
+            EnsureClause.StringIsNotNullOrEmpty(percentileAggregation.Field, percentileAggregation.Field, ExceptionMessage);
+
+            /// Median is Percentile(fieldName, 50)
+            percentileAggregation.KustoQL = string.Empty;
+
+            foreach (double percent in percentileAggregation.Percents)
+            {
+                percentileAggregation.KustoQL += $"['{percentileAggregation.Key}%{string.Format("{0:0.0}", percent)}']={KustoQLOperators.Percentile}({percentileAggregation.Field}, {percentileAggregation.Percents[0]}),";
```

Comment:
Maybe I wasn't clear in my comment
Here's a test you can add:
```
        [TestCase(ExpectedResult = "['_A%50.0']=percentile(fieldA, 50),['_A%60.0']=percentile(fieldA, 60)")]
        public string AggregationVisit_WithMultiplePercentileAgg_ReturnsPercentilePrimary()
        {
            var aggregateClause = new AggregationContainer()
            {
                PrimaryAggregation = new PercentileAggregation() { Field = "fieldA", Key = "_A", Percents = new double[] { 50, 60 }, Keyed = false },
            };

            var visitor = new ElasticSearchDSLVisitor(SchemaRetrieverMock.CreateMockSchemaRetriever());
            visitor.Visit(aggregateClause);

            return aggregateClause.KustoQL;
        }
```

It won't pass,  because of my comment.